website/docs/cloud-docs/workspaces/settings/health-assessments.mdx

```diff
@@ -0,0 +1,64 @@
+---
+page_title: Health Assessments - Workspaces - Terraform Cloud
+description: >-
+  Enable automatic jobs to check for lifecycle concerns, like whether your real-world infrastructure matches
+  your Terraform state file.
+tfc_only: true
+---
+
+# Health Assessments
+
+-> **Note:** Health Asessments (formerly Drift detection) is in beta and is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details and  [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve this feature.
```

Comment:
First - will this still be in beta when you merge these docs? I don't think so, but you know, good to check

```suggestion
-> **Note:** Health Assessments are available in the [Terraform Cloud **Business** tier](https://www.hashicorp.com/products/terraform/pricing). Please [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve this feature.
```